staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder.go

```diff
@@ -96,33 +96,39 @@ func BuildSwagger(crd *apiextensions.CustomResourceDefinition, version string) (
 
 	routes := make([]*restful.RouteBuilder, 0)
 	root := fmt.Sprintf("/apis/%s/%s/%s", b.group, b.version, b.plural)
+
+	var params []*restful.Parameter
+	nameParam := b.ws.PathParameter("name", "name of the "+b.kind).DataType("string")
+	namespaceParam := b.ws.PathParameter("namespace", "object name and auth scope, such as for teams and projects").DataType("string")
+
 	if b.namespaced {
-		routes = append(routes, b.buildRoute(root, "", "GET", "list", sampleList).
+		params = append(params, namespaceParam)
+		routes = append(routes, b.buildRoute(root, "", "GET", "list", sampleList, params).
```

Comment:
nit: could you make it one-liner ?

Reply:
how to one-line this?

Reply:
`routes=append(routes,b.buildRoute(root,"","GET","list",sampleList,params).Operation("list"+b.kind+"ForAllNamespaces"))`

at the moment there is a new line break right after the `.`, here: https://github.com/kubernetes/kubernetes/blob/db0c1917cae44577790aa941e1b15f3cf28088f1/staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder.go#L105

Reply:
updated